### PR TITLE
Add negative service account test

### PR DESCRIPTION
**Pull request by vatsalparekh**
**What this PR does / why we need it**:
This adds a missed test from https://github.com/kubevirt/kubevirt-ssp-operator/blob/master/functests/12-test-os-imaes-namespace_rbac.sh#L224-L230
**Which issue(s) this PR fixes**: 

Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: internal/operands/common-templates/constants.go [hunk 1]
```diff
@@ -11,6 +11,6 @@ const (
 	TemplateWorkloadLabelPrefix  = "workload.template.kubevirt.io/"
 	TemplateDeprecatedAnnotation = "template.kubevirt.io/deprecated"
 
-	CdiApiGroup = "cdi.kubevirt.io"
+	CdiApiGroup   = "cdi.kubevirt.io"
```
Comment (sarahbx) [inline internal/operands/common-templates/constants.go:14]: Let's drop this change, unrelated to pr

Comment (kwiesmueller) [reply internal/operands/common-templates/constants.go:14]: +1
I think gofmt should drop this anyways, don't we have a generation step for formating like in Kubevirt?

Comment (kwiesmueller) [reply internal/operands/common-templates/constants.go:14]: Once this is fixed and CI is green, lgtm from me.

File: tests/commonTemplates_test.go [hunk 2]
```diff
@@ -656,6 +656,18 @@ var _ = Describe("Common templates", func() {
 							},
 						}),
 				)
+				It("[test_id:4878]should not create any other resurces than the ones listed in the Edit Cluster role", func() {
```
Comment (shwetaap) [inline tests/commonTemplates_test.go:659]: We have duplicate entries of the test_id: 4876 - https://github.com/kubevirt/ssp-operator/blob/master/tests/commonTemplates_test.go#L638-L657. I thought the 4878 block was going to replace one of those duplicate blocks. But looks like it is not. Could you remove the duplicate entry?

File: internal/operands/common-templates/constants.go (hunk 1 above)
Comment (akrejcir) [reply internal/operands/common-templates/constants.go:14]: > +1
> I think gofmt should drop this anyways, don't we have a generation step for formating like in Kubevirt?

We do: `make fmt` and `make vet`, they are also run on any other build target.

Comment (vatsalparekh) [reply internal/operands/common-templates/constants.go:14]: This was done by either `make fmt` I guess, and it keeps coming back everytime, so I think it should be fixed

File: tests/commonTemplates_test.go (hunk 2 above)
Comment (vatsalparekh) [reply tests/commonTemplates_test.go:649]: Removed it now

File: internal/operands/common-templates/constants.go (hunk 1 above)
Comment (sarahbx) [reply internal/operands/common-templates/constants.go:14]: Looked at this locally, I see what it is doing now, lining up the `=`. 👍

**Review by sarahbx — APPROVED** [review]
/lgtm

**Pull request merged**